Documentation/troubleshooting.md

```diff
@@ -82,6 +82,24 @@ Note: The `ServiceMonitor` references a `Service` (not a `Deployment`, or a `Pod
 kubectl -n monitoring get secret prometheus-k8s -ojson | jq -r '.data["prometheus.yaml.gz"]' | base64 -d | gunzip | grep "my-service-monitor"
 ```
 
+### Debugging Why Resources Are Not Being Picked Up
```

Comment:
I'd move it before the `Has my `ServiceMonitor` been picked up by Prometheus?` section.
```suggestion
#### Is my `ServiceMonitor` rejected by Prometheus?
```